### PR TITLE
CHANGELOG: New release 1.0.2

### PR DESCRIPTION
**Pull request by ffmancera**
Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+ - Support multiconnect profiles. (2c60c834)
+ - Support bond vlan+srcmac tx hashing option. (ebf5777)
+ - Support routes created by iproute command. (f39f763)
+ - New property `Inetrface.COPY_FROM_MAC` on bridge and bond interfaces. (b089323)
+ - Support IEEE 802.1X support. (f9a1da3)
+
+### Bug fixes
+ - Fix removing OVS `external_ids` settings. (57e2547)
+ - Fix modification of OVS `external_ids` settings. (c99e708)
+ - Fix removing IPv6 default gateway. (fd21172)
```
Comment (cathay4t) [inline CHANGELOG:13]: - Improved the reliability on activating large amount of interfaces. (cbe5b57)

**Review by cathay4t — CHANGES_REQUESTED** [review]

File: CHANGELOG (hunk 1 above)
Comment (ffmancera) [reply CHANGELOG:13]: Done. Thanks!

File: CHANGELOG [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+ - New property `Inetrface.COPY_FROM_MAC` on bridge and bond interfaces. (b089323)
+ - Support IEEE 802.1X authentication. (f9a1da3)
+
+### Bug fixes
+ - Fix removing OVS `external_ids` settings. (57e2547)
+ - Fix modification of OVS `external_ids` settings. (c99e708)
+ - Fix removing IPv6 default gateway. (fd21172)
+ - Improved the reliability on activating large amount of interfaces. (cbe5b57)
+ - Fix creating a veth interface with an ethernet peer. (e17e70e)
+ - Remove SR-IOV VF profiles when reducing `total_vfs` in the PF. (f5f636f)
```
Comment (cathay4t) [inline CHANGELOG]: Please use `Ethernet.SRIOV.TOTAL_VFS` instead.

**Review by cathay4t — CHANGES_REQUESTED** [review]

File: CHANGELOG (hunk 2 above)
Comment (ffmancera) [reply CHANGELOG]: Done, thanks!

**Review by cathay4t — APPROVED** [review]
LGTM.

**Pull request merged**